_posts/2016-12-22-putting-down-puppet-3.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Putting down Puppet 3
+date: 2016-12-22
+github_username: nibalizer
+twitter_username: nibalizer
+---
+
+Puppet 3 is officially end of life on Jan 1, 2017. That is 8 days from today. Vox Pupuli's plan for deprecating Puppet 3 support in modules is as follows:
+
+* As of Jan 1, 2017, any module can accept Puppet 4 code on the master branch.
```

Comment:
I think that provides a clearer statement that we are officially and actively deprecating 3 in favor of 4